worlds/_bizhawk/__init__.py

```diff
@@ -10,7 +10,7 @@
 import enum
 import json
 import sys
-import typing
+from typing import Any, Optional, Sequence
```

Comment:
```suggestion
from typing import Any, Sequence
```
Optional isn't needed anymore since `| None` can be used now. Obviously you would have to modify the uses of Optional for you to be able to remove the import.